### PR TITLE
Fix disposing

### PR DESCRIPTION
**Pull request by dkoeb**
Had a look at all the classes that had a Console.WriteLine in their finalizer. All of them seem to be properly disposed now.

### CODE REVIEW

File: src/Clients/MainApp/FSpot/ThumbnailCache.cs [hunk 1]
```diff
@@ -48,13 +46,13 @@ public class ThumbnailCache : IDisposable {
 		#region Types
 		class Thumbnail {
 			// Uri of the image source
-			public SafeUri uri;
-	
+			public SafeUri Uri;
+
 			// The uncompressed thumbnail.
-			public Pixbuf pixbuf;
+			public Pixbuf Pixbuf;
```
Comment (decriptor) [inline src/Clients/MainApp/FSpot/ThumbnailCache.cs]: Can we convert these to Properties? Syntax just looks off at first glance otherwise.

Comment (decriptor) [inline src/Clients/MainApp/FSpot/ThumbnailCache.cs]: Does it make sense to keep this as a nested class?

File: src/Clients/MainApp/FSpot/ThumbnailCache.cs [hunk 2]
```diff
@@ -127,43 +125,28 @@ public void Dispose ()
...
-				pixbuf_mru.Clear ();
-			} 
-
+			pixbuf_mru.Clear ();
 		}
 
 		~ThumbnailCache ()
 		{
-			Console.WriteLine ("Finalizer called on {0}. Should be Disposed", GetType ());
 			Dispose (false);
```
Comment (decriptor) [inline src/Clients/MainApp/FSpot/ThumbnailCache.cs]: Dispose doesn't use 'disposing' anymore. Do we even need this? Or should 'disposing' be used?

**Comment by decriptor** [comment]
Other than the few comments it looks good.

**Comment by decriptor** [comment]
I think this looks good. I do this too, but I'm thinking it'd be better to split refactors (whitespace, naming, etc) into a separate commit so that it makes it easier to see the changes.

Thanks for the wook!

**Comment by dkoeb** [comment]
I prefer small and clean commits as well. It started out small and then grew and grew. In the end I thought about splitting it into more commits, but that would have taken a lot of time and I just wanted to get it done :-)
Next time I'm prepared...

**Comment by decriptor** [comment]
Been there done that. I know exactly what you are talking about :)

**Pull request merged**